compose.test.yml

```diff
@@ -3,5 +3,7 @@ services:
     build:
       target: development
     image: fcp-mpdp-frontend-development
```

Comment:
Don't think we need the image name

Reply:
Yeah it will.  I missed that when I was copying it across from the FCP version.

I can create a PR to fix that.